students/kosarzewski_maciej/lesson_02_flow_control/chocolate_bar.py

```diff
@@ -0,0 +1,8 @@
+x = int(input())
+y = int(input())
+part = int(input())
+surface = x*y
```

Comment:
It's not stated in PEP8 but generally it's recommended to use space between math operators
'surface = x * y'